### PR TITLE
Gho 201

### PR DESCRIPTION
**Pull request by attiks**

### CODE REVIEW

**Review by left23 — APPROVED** [review]
I've removed the stylelint `no-browser-hacks` plugin as per https://humanitarian.atlassian.net/browse/OPS-7558?focusedCommentId=129218 and I can run `sass:build` locally.
And re-arranged the packages in `package.json` to match https://humanitarian.atlassian.net/browse/CD-328

I'll leave the rest of the testing to others who are more familiar with the implementation details.

**Review by lazysoundsystem — COMMENTED** [review]
Builds okay locally and all looks right, except there's an issue that might be related to the node-title issue:
image
Also, on that page, I can't click any buttons on the site because the footer is 'above' them - when I lower the z-index of the footer, I can click, e.g. 'edit'.

My local url for that page: https://gho.test/monthly-updates/global-humanitarian-overview-march-update

I tried rebuilding sass in the subtheme - anything else I might have missed.

@left23 do you have any suggestions?

**Comment by left23** [comment]
@lazysoundsystem I think the screenshot depicts what happens when the image doesn't load. I can get the same result on Prod https://gho.unocha.org/monthly-updates/global-humanitarian-overview-march-update if I remove the image via the web dev tools.

Once this is on the dev site there's the option of running VRT. That should avoid issues with missing images on local sites.

File: html/themes/custom/common_design_subtheme/common_design_subtheme.libraries.yml [hunk 1]
```diff
@@ -100,11 +100,6 @@ gho-aside:
   dependencies:
     - common_design_subtheme/gho-caption
 
-gho-hero-image:
```
[the remark this answers is not in the document]
Comment (attiks) [reply html/themes/custom/common_design_subtheme/common_design_subtheme.libraries.yml:103]: Key exists twice, removed duplicate

**Review by lazysoundsystem — APPROVED** [review]
Thanks for the explanation, @left23 - removing the image on prod also makes the footer div cover the local task links.

So approval from me too.

**Pull request merged**